<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerConfigWidget</class>
 <widget class="QWidget" name="PlayerConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>587</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_grpGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Seconds to &amp;jump on seek:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_SeekJumpLength</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="kcfg_StepJumpLength">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_PlayerBackend"/>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="kcfg_ShowPositionTimeEdit">
        <property name="text">
         <string>Show editable position control</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;VideoPlayer backend:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_PlayerBackend</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="kcfg_SeekJumpLength">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fra&amp;me step amount:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_StepJumpLength</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="kcfg_VideoAutoPlay">
        <property name="text">
         <string>Start playing video when loaded</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="kcfg_UnpauseOnDoubleClick">
        <property name="text">
         <string>Resume playback after line double click</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_grpSubtitles">
     <property name="title">
      <string>Subtitles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Outline color:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_OutlineColor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KColorCombo" name="kcfg_FontColor"/>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="kcfg_FontFamily"/>
      </item>
      <item row="3" column="1">
       <widget class="KColorCombo" name="kcfg_OutlineColor"/>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Outline width:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_OutlineWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Te&amp;xt color:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_FontColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Font family:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_FontFamily</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Font si&amp;ze:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_FontPointSize</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="kcfg_OutlineWidth">
        <property name="suffix">
         <string notr="true"> px</string>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_FontPointSize">
        <property name="suffix">
         <string notr="true"> px</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="kcfg_Antialias">
        <property name="text">
         <string>Text Antialiasing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_grpPreview">
     <property name="title">
      <string>Subtitles Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QWidget" name="m_layeredWidget" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="TextOverlayWidget" name="m_textOverlayWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>TextOverlayWidget</class>
   <extends>QWidget</extends>
   <header>textoverlaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_PlayerBackend</tabstop>
  <tabstop>kcfg_VideoAutoPlay</tabstop>
  <tabstop>kcfg_UnpauseOnDoubleClick</tabstop>
  <tabstop>kcfg_SeekJumpLength</tabstop>
  <tabstop>kcfg_StepJumpLength</tabstop>
  <tabstop>kcfg_ShowPositionTimeEdit</tabstop>
  <tabstop>kcfg_FontFamily</tabstop>
  <tabstop>kcfg_FontPointSize</tabstop>
  <tabstop>kcfg_FontColor</tabstop>
  <tabstop>kcfg_OutlineColor</tabstop>
  <tabstop>kcfg_OutlineWidth</tabstop>
  <tabstop>kcfg_Antialias</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
